osdk/src/base_crate/riscv64.ld.template

```diff
@@ -0,0 +1,55 @@
+OUTPUT_ARCH(riscv)
+ENTRY(_start)
+KERNEL_LMA = 0x80200000;
```

Comment:
Doesn't `0x8020000` work?

Reply:
I tried, but sadly it didn't work. I haven't investigated it in detail, since it may need to analyze SBI's specification or implementation.